tests/test_study/test_study_functions.py

```diff
@@ -18,13 +18,13 @@ def test_get_study(self):
         self.assertEqual(len(study.setups), 30)
 
     def test_get_tasks(self):
-        study_id = 14
+        study_id = 1
```

Comment:
Could you briefly elaborate the issue with study 14?

Reply:
I removed it from the test server, and replaced it by study 1